nf_core/pipeline-template/nextflow.config

```
@@ -233,6 +234,10 @@ dag {
     enabled = true
     file    = "${params.outdir}/pipeline_info/pipeline_dag_${trace_timestamp}.html"
 }
+prov {
+    enabled = true
+    file = "${params.outdir}/pipeline_info/manifest_${trace_timestamp}.json"
```

I think it could be good to include `bco` in the filename, to make it clear that it's BCO format. Especially if we add more manifest JSON files in the future.

```suggestion
    file = "${params.outdir}/pipeline_info/manifest_${trace_timestamp}.bco.json"
```